components/overflow-group/overflow-group.js

```diff
@@ -61,104 +50,15 @@ function createMenuItemSeparator() {
 }
 
-function createMenuItemMenu(node) {
-	const menuOpener =
-		node.querySelector('d2l-dropdown-button')
-		||  node.querySelector('d2l-dropdown-button-subtle');
-
-	const openerText = node.text || menuOpener.text;
-	const disabled = !!node.disabled;
-	const subMenu = node.querySelector('d2l-menu');
-
-	const subItems = Array.from(subMenu.children).map((node) => convertToDropdownItem(node));
-
-	return html`<d2l-menu-item
-		?disabled=${disabled}
-		text="${openerText}">
-		<d2l-menu>
-			${subItems}
-		</d2l-menu>
-	</d2l-menu-item>`;
-}
-
-function convertToDropdownItem(node) {
-	const tagName = node.tagName.toLowerCase();
-	switch (tagName) {
-		case 'd2l-button':
-		case 'd2l-button-subtle':
-		case 'button':
-		case 'd2l-button-icon':
-		case 'd2l-selection-action':
-			return createMenuItem(node);
-		case 'a':
-		case 'd2l-link':
-			return createMenuItemLink(node);
-		case 'd2l-menu':
-		case 'd2l-dropdown':
-		case 'd2l-dropdown-button':
-		case 'd2l-dropdown-button-subtle':
-		case 'd2l-dropdown-context-menu':
-		case 'd2l-dropdown-more':
-		case 'd2l-selection-action-dropdown':
-			return createMenuItemMenu(node);
-		case 'd2l-menu-item':
-		case 'd2l-selection-action-menu-item':
-			// if the menu item has children treat it as a menu item menu
-			if (node.children.length > 0) {
-				return createMenuItemMenu(node);
-			} else {
-				return createMenuItem(node);
-			}
-	}
-	if (node.getAttribute('role') === 'separator') {
-		return createMenuItemSeparator();
-	}
-}
-
 /**
  *
  * A component that can be used to display a set of buttons, links or menus that will be put into a dropdown menu when they no longer fit on the first line of their container
  * @slot - Buttons, dropdown buttons, links or other items to be added to the container
- * @fires d2l-overflow-group-updated - Dispatched when there is an update performed to the overflow group
 */
-class OverflowGroup extends RtlMixin(LocalizeCoreElement(LitElement)) {
+class OverflowGroup extends OverflowGroupMixin(LitElement) {
 
 	static get properties() {
 		return {
-			/**
-			 * Use predefined classes on slot elements to set min and max buttons to show
-			 * @type {boolean}
-			 */
-			autoShow: {
-				type: Boolean,
-				attribute: 'auto-show',
-			},
-			/**
-			 * minimum amount of buttons to show
-			 * @type {number}
-			 */
-			minToShow: {
-				type: Number,
-				reflect: true,
-				attribute: 'min-to-show',
-			},
-			/**
-			 * maximum amount of buttons to show
-			 * @type {number}
-			 */
-			maxToShow: {
-				type: Number,
-				reflect: true,
-				attribute: 'max-to-show',
-			},
-			/**
-			 * Set the opener type to 'icon' for a `...` menu icon instead of `More actions` text
-			 * @type {'default'|'icon'}
-			 */
-			openerType: {
-				type: String,
-				attribute: 'opener-type'
-			},
 			/**
 			 * Setting this property will change the style of the overflow menu opener
 			 * @type {'default'|'subtle'}
```

Comment:
`openerStyle` currently only applies to `d2l-overflow-group` and not to the future `d2l-filter-overflow-group` so I've left it only here.